<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1623</width>
    <height>557</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test-External-Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>150</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Robot Data Viewer</string>
    </property>
    <widget class="QTextEdit" name="textPrint">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>521</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_CLEAR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BTN_TEST">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Motion Test</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GB_NETWORK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Network Connection</string>
    </property>
    <widget class="QLineEdit" name="LE_Ip">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
     <property name="maxLength">
      <number>32367</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_CmdPort">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>5000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Server</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>70</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cmd Server</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_DataPort">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>5001</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_CmdConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_DataConnect">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Cobot Initialize</string>
    </property>
    <widget class="QProgressBar" name="PGB_Init">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_CobotInit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BTN_GET_JOINT_AND_TCP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Joint and TCP</string>
    </property>
   </widget>
   <widget class="QFrame" name="FRAME_STATUS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>630</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HS_BASE_SPEED">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>40</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_BASE_SPEED">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_JOINT_LIST">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_TCP_LIST">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_MOTION_PAUSE">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 233, 79);</string>
    </property>
    <property name="text">
     <string>Motion
Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_MOTION_HALT">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>80</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 175, 62);</string>
    </property>
    <property name="text">
     <string>Motion
Halt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_MOTION_RESUME">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string>Motion
Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_COLLISION_RESUME">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>80</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string>Collision
Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_MANUAL_SEND">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>222</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual_Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MANUAL_SEND">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="FRAME_STATUS_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>630</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="BTN_START_SCAN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Start Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_GO_READY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Go Ready</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
